CodeStarterCamp_Week2/main.swift

```diff
@@ -8,5 +8,43 @@
 
 import Foundation
 
-print("Hello, World!")
+//찍은 번호
+let myLottoNumbers: [Int] = [4, 13, 24, 36, 38, 45]
 
+//로또 번호 배열
+var lottoNumbers: [Int] = []
+
+//로또 번호 랜덤 뽑기
+func pickLottoNumbers() {
+    while lottoNumbers.count < 6 {
+        let numbers = Int.random(in: 1...45)
+        if !lottoNumbers.contains(numbers) {
+            lottoNumbers.append(numbers)
+        }
+    }
+}
+
+pickLottoNumbers()
+
+//찍은 번호와 로또 번호가 일치하면 들어가는 배열
+var coincideNumbers: [Int] = []
+
+func matchNumber(_myLottoNumbers: [Int], lottoNumbers: [Int]) {
+    for num in myLottoNumbers {
+        if lottoNumbers.contains(num) {
+            coincideNumbers.append(num)
+        }
+    }
+}
+
+matchNumber(_myLottoNumbers: myLottoNumbers, lottoNumbers: lottoNumbers)
+
+func comfirmNumber() {
+    if coincideNumbers.isEmpty {
```

Comment:
최종 결과 출력 부분 메서드 아주 로직이 깔끔하네요. 👍🏻 

다만, 결국 확인하는 건 하나의 숫자가 아니라 `여러 숫자들` 이죠!?
내부 `if문`에서 체크하는 것도 숫자들의 배열이구요. 😄 

컨셉을 정확하게 유지하려면, 메서드의 네이밍도 `복수형`으로 변경되어야 할 것 같습니다!
참 어렵죠 네이밍? ㅎㅎㅎ..

자신만의 기준을 갖고 `일관성`을 유지하는 게 중요하겠습니다!!

```suggestion
func comfirmNumbers() {
    if matchedNumbers.isEmpty {
```

Reply:
수정했습니다!